src/mpid/ch3/src/mpid_vc.c

```diff
@@ -326,7 +326,7 @@ int MPIDI_GPID_ToLpidArray( int size, MPIDI_Gpid in_gpid[], MPIR_Lpid lpid[] )
 		   an error message and its not appropriate to use the
 		   regular error code system */
 		/* printf("No matching pg foung for id = %d\n", pgid ); */
-		lpid[i] = -1;
+		lpid[i] = (uint64_t) - 1;
```

Comment:
I thought I have made `MPIR_Lpid` signed already. What was the warning?

Reply:
Let me see if this is gone now.

Reply:
Removed.